README.md

```diff
@@ -66,6 +66,24 @@ Similarly to recording videos, we have a script to measure the performance of a
 
 ## Developing
 
+To install dependencies, first run `npm run bootstrap`. This command will populate the repos root `/node_modules` and underyling packages `/node_modules`. From there, you have a few options depending on where you'd like to make changes.
+
+### Webviz Core
+
+```npm run webviz-dev```
+
+### Docs
+
+```npm run docs```
+
+### Storybook
+
+```npm run storybook```
+
+Useful for `regl-worldview` changes.
+
+### Full list of commands
```

Comment:
These are called `scripts` in the npm :) Technically we have other scripts as well, maybe change to `Other scripts`?

Reply:
Will do